WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java

```diff
@@ -104,9 +103,7 @@
     private static final int SELECT_LIBRARY_MENU_POSITION = 100;
     private static final int CLEAR_FEATURED_IMAGE_MENU_POSITION = 101;
 
-    private PostModel mPost;
-    private SiteModel mSite;
-
+    private PostSettingsListener mListener;
```

Comment:
I don't fully agree with what you're saying, but it makes no difference to me. So, I am happy to go with anything. So, I guess we are going with `EditPostActivityHook` in this case.